README.md

```diff
@@ -18,9 +18,9 @@ To learn more about the contents of this repository, see this README and the REA
 
 ## Requirements
 
-This repository requires Yarn v3+ and Node 16.10+.
-
-To install Yarn v3, see [Yarn Installation](https://yarnpkg.com/getting-started/install).
+- Node 16.10+
+- React 16.14+
+- Yarn v3+ is required only at the root level of this repo and when developing examples and packages in this repo. See [Yarn Installation](https://yarnpkg.com/getting-started/install). External consumption of the React SDK does not require Yarn.
```

Comment:
WDYT of something like this:

```
Yarn v3+ is only required when working with this repo. See [Yarn Installation](https://yarnpkg.com/getting-started/install).
```

the reason i think this is OK is the nature of package managers and how developers interact with them. it's known that you can use any package manager to consume packages on NPM, but may need to use a specific one when working within a repo.